resim/msg/fuzz_helpers.hh

```diff
@@ -164,6 +186,24 @@ Detection3DArray random_element(
   return result;
 }
 
+template <typename Rng>
+Detection2DArray random_element(
+    TypeTag<Detection2DArray> /*unused*/,
+    InOut<Rng> rng) {
+  Detection2DArray result;
+  result.mutable_header()->CopyFrom(random_element<Header>(rng));
+
+  constexpr int MIN_ELEMENTS = 1;
+  constexpr int MAX_ELEMENTS = 10;
```

Comment:
I'm always somewhat averse to randomizing numbers of elements in lists, and not sure what the right way to do this is. (The main issue is that if you have an unlucky seed, you end up only testing 1 element.)

Reply:
I'll increase MIN_ELEMENTS to at least 5! 😀